node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/task/util/yum/YumCommand.java

```diff
@@ -47,11 +48,18 @@ public T enableRepo(String... repo) {
         return getThis();
     }
 
+    /** Enables the given repos for this command */
+    public T disableRepo(String... repo) {
+        disabledRepos = List.of(repo);
+        return getThis();
+    }
+
     protected abstract T getThis(); // Hack to get around unchecked cast warning
 
     protected void addParametersToCommandLine(CommandLine commandLine) {
         commandLine.add("--assumeyes");
         enabledRepos.forEach(repo -> commandLine.add("--enablerepo=" + repo));
+        disabledRepos.forEach(repo -> commandLine.add("--disablerepo=" + repo));
```

Comment:
Move disableRepos before enableRepos.  I have only seen this order, and don't know if this order will work.  Unless you know it's OK to put disable last?